codes/quantum/qubits/4d_hyperbolic.yml

```diff
@@ -0,0 +1,40 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: four_dimensional_hyperbolic
+physical: qubits
+logical: qubits
+
+name: 'Four-dimensional hyperbolic code'
+introduced: '\cite{arXiv:1310.5555}'
+
+description: |
+  There exists \(\epsilon\), a four-dimensional hyperbolic manifold \(M\) and a sequence of manifolds \(M_i\) such that
```

Comment:
Start of this is a bit jarring. First sent should be what the code is based on. How do the manifolds feature in the code construction?